profile/README.md

```diff
@@ -5,3 +5,104 @@
     \__, /_/ /_/\____/      Social:    <a href="https://twitter.com/_gnoland">Twitter</a> - <a href="https://discord.gg/tF2X8M6cVj">Discord</a> - <a href="https://reddit.com/r/gnoland">Reddit</a> - <a href="https://youtube.com/@_gnoland">YouTube</a>
    /____/                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 </pre>
+
+***
+
+## Why Gno? 
+
+- Interpreted Gnolang: Gno.land utilizes the Gnolang programming language, which is based on Go. It is executed through a specialized virtual machine called the GnoVM, purpose-built for the Gno.land blockchain. While Gnolang shares similarities with Go in terms of syntax, it currently lacks go routine support. However, this feature is planned for future development, ensuring deterministic GnoVM executions.
+
+- Consensus Protocol - Tendermint2: Gno.land achieves consensus between blockchain nodes using the Tendermint2 consensus protocol. This approach ensures secure and reliable network operation.
+
+- Inter-Blockchain Communication (IBC): In the future, Gno.land will be able to communicate and exchange data with other blockchain networks within the Cosmos ecosystem through the Inter-Blockchain Communication (IBC) protocol.
+
+## Why Go-Based?
+The decision to base Gno.land's language on Go was influenced by the following factors:
+
+- Standard and Secure Language: Go is a well-established and secure programming language, widely adopted in the software development community. By leveraging Go's features, Gno.land benefits from a robust and proven foundation.
+
+- User-Friendly: Go's simplicity and ease of understanding make it beginner-friendly. This accessibility lowers the entry barrier for developers to create Smart Contracts on the Gno.land platform.
```

Comment:
Please let's sync this question with the one on the [FAQ's](https://github.com/gnolang/awesome-gno/blob/main/FAQ.md), if not word-by-word at least in spirit. (That one can be improved too, but I'm just saying that we should combine the two things.)

```suggestion

- An established language: using Go instead of creating a new programming language enables existing Go developers to bring over their knowledge to Gno, making adopting the language not an intrinsic time investment on the success of the blockchain. Gno re-uses the syntax and a majority of the standard libraries, adapts Go's package model and adapts it for the blockchain context.
- Security through the package model: by leveraging Go's existing package model, which distinguishes between exported and unexported symbols, realms (smart contracts) are capable of simply discriminating what functionality, types and data should be available for consumption outside of the realm. By enforcing all on-chain published code to be open source and creating a public on-chain registry of published re-usable packages, Gno realms (smart contracts and packages can be easily audited and build upon each other's reputation.
```

Reply:
Sure, though this PR predates the FAQ